lib/vsc/mympirun/mpi/mpi.py

```diff
@@ -151,8 +151,37 @@ def which(cmd):
     return None
 
 
-class MPI(object):
+class RunFileLoopMPI(RunFile, RunLoop):
+    """
+    RunFile specific for MPI because intel doesn't feel like fixing their shit
+    """
+    def __init__(self, cmd, **kwargs):
+        self.options = kwargs.pop('options', None)
```

Comment:
actually, just grab `filename` , but do pass it down (don't pop it from `kwargs`):

```python
self.timeout = kwargs.pop('timeout', None)
self.filename = kwargs.get('filename', None)
```